lib/exercise_cases.rb

```diff
@@ -1,2 +1,30 @@
 require 'ostruct'
 require 'json'
+
+class ExerciseCase < OpenStruct
+  def name
+    'test_%s' % description.gsub(/[^\w ?!]/, '').gsub(/[- ]/, '_')
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  protected
+
+  def raises_error?
+    expected.to_i == -1
+  end
+
+  def assert_or_refute
```

Comment:
Future:
It may be possible to add logic into the `assert` method that works out the correct assert to use including `assert_equal` or `assert_error`